tests/containers/buildah_image.pm

```diff
@@ -37,7 +37,11 @@ sub run {
         if (check_os_release('suse', 'PRETTY_NAME')) {
             # sle15-working-container is the default name given to a container. it is created in test_container_image
             test_opensuse_based_image(image => 'sle15-working-container', runtime => 'buildah');
-            allow_selected_insecure_registries(runtime => 'podman');
+            # Due to the steps from the test_opensuse_based_image previously,
+            # the image has been committed as refreshed
+            test_containered_app(runtime => 'buildah',
+                dockerfile => 'Dockerfile.test',
+                base       => 'refreshed');
```

Comment:
It is possible that `refreshed` image won't exist. Can you use `sle15-working-container` instead?
```suggestion
            test_containered_app(runtime => 'buildah',
                dockerfile => 'Dockerfile.test',
                base       => 'sle15-working-container');
```

Reply:
i cant because the `sle15-working-container` appears as a container. So the `test_containered_app` cant find it in the repo and cant use it. The `refreshed` is the derived image from it. 
i have resolved another similar comment. Thing is that as it is, it will not work for opensuse. and there should enable it in another PR. then we have to think how to improve this. Do you agree?

Reply:
Please create a ticket for that as well…